package.json

```diff
@@ -42,6 +42,7 @@
     "testEnvironment": "node",
     "testPathIgnorePatterns": [
       "/dist/",
+      "<rootDir>/packages/xo-server-test",
```

Comment:
Wouldn't `/xo-web/` also need `<rootDir>/packages`?

Reply:
For what reason?

Reply:
My bad is bellow!

Reply:
For the same reason you're using it here: "to prevent it from accidentally ignoring all of your files in different environments that may have different root directories". What's the difference between `xo-web` and `xo-server-test`?

Reply:
I'm agree with you, it's a mistake :+1: